consensus/ibft/src/integration-test/java/tech/pegasys/pantheon/consensus/ibft/tests/RoundChangeTest.java

```diff
@@ -149,19 +148,17 @@ public void whenSufficientRoundChangeMessagesAreReceivedForNewRoundLocalNodeCrea
     final RoundChange rc3 = peers.getNonProposing(2).injectRoundChange(targetRound, empty());
     final RoundChange rc4 = peers.getProposer().injectRoundChange(targetRound, empty());
 
-    final NewRound expectedNewRound =
-        localNodeMessageFactory.createNewRound(
+    final Proposal expectedNewRound =
```

Comment:
nit: this should be renamed to expectedProposal

Reply:
done (plus lots of others).